legato-api/pom.xml

```diff
@@ -7,17 +7,28 @@ terms of the Eclipse Public License v1.0 which accompanies this distribution,
 and is available at http://www.eclipse.org/legal/epl-v10.html INTERNAL
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <parent>
+
+<!--  <parent>
```

Comment:
In general we should avoid crud in our code. If this part of the configuration is not required, it is better to remove it. Git will take care of recovering it if needed.